qmlti/src/lti/view/student_nav.php

```diff
@@ -1,26 +1,48 @@
-<div id="body" class="container-fluid col-md-6">
-<h1>Student Portal</h1>
-<form action="student_nav.php" method="POST">
-  <table class="DataTable table table-sm" cellpadding="0" cellspacing="0">
-    <tr class="GridHeader">
-      <th>Assessment Name</th>
-      <th>Launch</th>
-      <th># of Possible Attempts</th>
-      <th># of Attempts Taken</th>
-    <?php if ($bool_coaching_report) { ?>
-      <th>Coaching Report</th>
-    <?php } ?>
-    </tr>
-    <tr class="GridRow">
-      <td><?php echo $assessment->Session_Name; ?></td>
-      <td><?php echo $launch; ?></td>
-      <td><?php echo $parsed_attempts; ?></td>
-      <td><?php echo $past_attempts; ?></td>
-    <?php if ($bool_coaching_report) { ?>
-      <td><input class="btn btn-sm btn-link" type="submit" name="action" value="View Coaching Report" formtarget="_blank"/></td>
-    <?php } ?>
-    </tr>
-  </table>
-</form>
-<br>
-</div>
+<body class="bg-pattern" style="background-color: rgb(250, 250, 250);">
+  <div id="Wrapper">
+    <div id="MainContentWrapper" class="col-md-6 col-md-offset-3" style="padding-top: 5em">
+      <div id="ContentWrapper">
+        <div id="HeaderWrapper" class="header-top padding-top-md">
+          <a href="https://www.questionmark.com/"><img id="logoImage" src="/web/images/logo.png" alt="Questionmark" class="center-block" /></a>
+        </div>
+        <hr class="qm-divider" />
+        <div id="PageContent" class="block-color">
+          <div class="container-fluid"><p><button type="button" class="btn btn-default btn-grey no-margin" onclick="location.href='<?php echo $return_url; ?>'">Back to Course</button></p></div>
+            <div id="body" class="container-fluid" style="padding: 0px">
```

Comment:
This div isn't being closed

Reply:
And everything inside this div needs to be indented once more.

Reply:
Made the indentation, but I'm not seeing the unclosed Div. I've cleaned up the HTML to make it a bit clearer though, there was inconsistencies in the page.